src/_pytest/pathlib.py

```diff
@@ -636,9 +632,39 @@ def _import_module_using_spec(
 
     if spec_matches_module_path(spec, module_path):
         assert spec is not None
+        # Attempt to import the parent module, seems is our responsibility:
+        # https://github.com/python/cpython/blob/73906d5c908c1e0b73c5436faeff7d93698fc074/Lib/importlib/_bootstrap.py#L1308-L1311
+        parent_module_name, _, name = module_name.rpartition(".")
```

Comment:
`parent_module_name` can be empty if `module_name` doesn't contain a dot, right? If so I would avoid searching it in `sys.modules` just in case

Reply:
Good catch, thanks!